Makefile

```diff
@@ -0,0 +1,38 @@
+DOCKER_IMG = klakegg/hugo:ext-alpine
+DRAFT_ARGS = --buildDrafts --buildFuture  --buildExpired
+
+# Keep these consistent with the values in config.yaml and layouts/index.redirects:
+NEXTv=v3.5
+LATESTv=v3.5
```

Comment:
If you change both of these to v2, then the link checking should work.

Reply:
Changed, will reintroduce the link check in the next commit to test.